source/JsMaterialX/JsMaterialXCore/JsDocument.cpp

```diff
@@ -59,8 +60,8 @@ EMSCRIPTEN_BINDINGS(document)
         .function("getTypeDefs", &mx::Document::getTypeDefs)
         .function("removeTypeDef", &mx::Document::removeTypeDef)
         BIND_MEMBER_FUNC("addNodeDef", mx::Document, addNodeDef, 0, 3, stRef, stRef, stRef)
-        BIND_MEMBER_FUNC("addNodeDefFromGraph", mx::Document, addNodeDefFromGraph, 7, 7, const mx::NodeGraphPtr, 
-            stRef, stRef, stRef, bool, stRef, std::string)
+        BIND_MEMBER_FUNC("addNodeDefFromGraph", mx::Document, addNodeDefFromGraph, 4,4, const mx::NodeGraphPtr, const std::string,
```

Comment:
Should these arguments be const references to strings rather than const string values?

Reply:
Fixed!